mobile_payment/utils.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+import requests
+import logging
+from datetime import datetime, timedelta
+from django.conf import settings
+from django.utils.translation import gettext_lazy as _
+import json
+from .models import ApiUtilitie
+import threading
+
+logger = logging.getLogger(__name__)
+
+lock = threading.Lock()
+
+def access_token():
+    # make it thread safe
+    with lock:
+        try:
+            NOW = datetime.now()
+            api_name = 'Qmoney_token'
+            access_token = get_access_token(api_name)
+            return {"Authorization": f"Bearer {access_token}"}
+        except Exception as e:
+            logging.exception("failed to get access token")
+            return e
+
+
+def get_access_token(api_name):
+    api_utility = ApiUtilitie.objects.get(name=api_name)
+    current_token = api_utility.access_token
+    token_expiry_date = api_utility.access_TokenExpiry
+
+    if token_expiry_date is None or token_expiry_date <= datetime.now():
```

Comment:
I don't think you should provide as a config the initial token as it can be retrieved programmatically. So please remove it from the config.

Reply:
what should i remove exactly pls